### PR TITLE
Update Readme to match the Cross-platform structure.

### PR DESCRIPTION
**Pull request by miguelandres**
- Add table of repo status.
- Update intro and description of what a variable is
- Remove roadmap and state of development links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,27 @@
...
+Remixer allows you to easily define UI variables in your app that can be adjusted while it is running. This can help you and your team to iterate very quickly on design issues because you don't need to rebuild or even restart your app to see the changes reflected.
 
-The Remixer SDK is currently under development and will be available for iOS, Android and Web. Please review the State of Development table for status of each platform.
+Remixer is available in three platforms ([Android](http://github.com/material-foundation/material-remixer-android), [iOS](http://github.com/material-foundation/material-remixer-ios) and [Web](http://github.com/material-foundation/material-remixer-js)) and comes with a helpful [Remote Controller](http://github.com/material-foundation/material-remixer-remote-web) that lets you interact with variables from another device.
```
Comment (chriscox) [inline README.md]: Can you change web to JavaScript

Comment (miguelandres) [inline README.md]: Sure thing

Comment (chuga) [inline README.md]: "and all of them support setting up a Remote Controller that lets you change variables from a web dashboard" (or web controller). We shouldn't say from another device as that is confusing.

File: README.md [hunk 2]
```diff
@@ -1,45 +1,27 @@
...
+| [Remote Controller](http://github.com/material-foundation/material-remixer-remote-web) | [![Build Status](https://travis-ci.org/material-foundation/material-remixer-remote-web.svg?branch=develop)](https://travis-ci.org/material-foundation/material-remixer-remote-web) |  | [![npm version](https://badge.fury.io/js/material-remixer-remote-web.svg)](https://badge.fury.io/js/material-remixer-remote-web) |
 
-- **Booleans**
-	- Example: Use a boolean variable to hide or show a view during development.
+## Variables
 
-- **Strings**
-	- Example: Use a string variable to easily try different options for a header.
+A Remixer variable has a unique key associated with it, and can be a Boolean, Color, Number (represented as floats) or String.
```
Comment (chuga) [inline README.md]: "... and can be of type Boolean, Color, Number or String" I'd remove the part about floats from here, it should be mentioned in each repo though.

File: README.md [hunk 3]
```diff
@@ -1,45 +1,27 @@
...
 
-Remixer also supports setting a list of values a variable can be set to. For example, a color variable can be configured so that its value can only be changed to red, blue or yellow.
+Remixer lets you define a range of values (for variables of type Number), or limit values to members of a list you provide. This lets you ensure that values always _make sense_.
```
Comment (chuga) [inline README.md]: "Variables support being constrained to a limited set of values to give you better control of what can change. Numeric variables can also be configured as continuous or discrete ranges."

File: README.md [hunk 4]
```diff
@@ -1,45 +1,27 @@
...
-
-## Roadmap and State of development
-
-Development of Remixer is ongoing across the supported platforms. Visit our [State of Development](https://github.com/material-foundation/material-remixer/wiki/State-of-Development) wiki which details the status of development for each platform. In addition, visit our current [Roadmap](https://github.com/material-foundation/material-remixer/wiki/Roadmap) wiki for a list of features planned for future releases.
-
-## Repositories
-
-Platform specific libraries and tools can be found in the following GitHub repos:
+A separate project, the [Remote Controller](http://github.com/material-foundation/material-remixer-remote-web), is provided for those times when by interacting with Remixer you may make it hard to see the UI you're adjusting. The Remote Controller lets you adjust variable values in a separate device so you can properly appreciate all changes.
```
Comment (chuga) [inline README.md]: "Remixer can be configured to use a Remote Controller []. In this mode your variables can be changed from outside the app, allowing you to collaborate with other people and keep your UI free of overlays. Make sure to check out our Firebase Remote Controller repo [] to learn more about this feature."

File: README.md [hunk 5]
```diff
@@ -1,45 +1,27 @@
...
 
-- [iOS](https://github.com/material-foundation/material-remixer-ios) - Remixer for iOS.
-- [Android](https://github.com/material-foundation/material-remixer-android) - Remixer for Android.
-- [JavaScript](https://github.com/material-foundation/material-remixer-js) - Remixer for JavaScript.
-- [Web Remote](https://github.com/material-foundation/material-remixer-remote-web) - Remixer web remote controller for all platforms.
+Since there is a bit of platform specific setup in each case, look at the documentation in your platform's Remixer Repo  and the Remote Controller's for more information.
```
Comment (chuga) [inline README.md]: "For details on how to set up this mode in your app, please visit one of the platform specific repos."

File: README.md (hunk 1 above)
Comment (miguelandres) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (miguelandres) [inline README.md]: Done

File: README.md (hunk 3 above)
Comment (miguelandres) [inline README.md]: _Done_

File: README.md (hunk 4 above)
Comment (miguelandres) [inline README.md]: Done

File: README.md (hunk 5 above)
Comment (miguelandres) [inline README.md]: Done

**Pull request merged**